### ISSUE TITLE
Ability to set default size for scratchpad windows

### ISSUE THREAD

**Issue by sujay1844** [swaywm/sway#6294]
As you see the title explains everything.
Is there a way to change the default size of scratchpad windows??
If I'm not wrong, the current default resolution is `86x35`. I want to change it to something like `135x35`.
Although I can manually change it once it's in the scratchpad but when I quit the window or reboot the computer, it changes back to `86x35`.
Has this feature been implemented??
Thanks in advance!! By the way, Sway is great!!

**Comment by eigenbrot** [comment]
I agree that a default size would be great, but the same effect can be achieved through a simple modification of the config file. In your example:
```
# Open the scratch pad and resize it
bindsym $mod+Shift+a scratchpad show, resize 135x35
```
This has a nice benefit of ensuing the scratch pad has the same aspect ratio on monitors with different scale factors. A downside is that all scratchpad windows will be affected, but I suppose you could have another key combo to cycle them without resizing.

**Comment by covert8** [comment]
When using this workaround with percentiles it gets applied to another window when i close the scratch pad with the same command

**Comment by stelcodes** [comment]
Yeah I can confirm that the solution from @eigenbrot does not work since it can resize a random window when the scratchpad is closed. It would be nice to get a default size option since this seems like it requires a script which queries the window node tree in order to enforce a scratchpad size.
